src/main/scala/mesosphere/marathon/core/health/HealthCheckShield.scala

```diff
@@ -0,0 +1,8 @@
+package mesosphere.marathon
+package core.health
+
+import mesosphere.marathon.state.Timestamp
+import mesosphere.marathon.core.task.Task
+
+case class HealthCheckShield(taskId: Task.Id, ttl: Timestamp) {
```

Comment:
Another cosmectic point: to me ttl refers to a duration, but at this point, you changed it to be now+ttl, so it's not a TTL anymore. Completely optional, but maybe we should find a better suited name for this field. Eg: expiryTime, endTime, ...